packages/async-flow/src/replay-membrane.js

```diff
@@ -605,6 +614,7 @@ export const makeReplayMembraneForTesting = ({
    * These are the only ones that are driven from the interpreter loop
    */
   const topDispatch = harden({
+    [START_GENERATION_OP_NAME]: startGeneration,
     doFulfill,
     doReject,
     // doCall, // unimplemented in the current plan
```

Comment:
These lines are surprisingly asymmetric. How about just 
```suggestion
    startGeneration,
    doFulfill,
    doReject,
    // doCall, // unimplemented in the current plan
```
? 

Alternatively, change the others to follow your first line convention, which would at least be consistent. I prefer the briefer form though.

Reply:
Made briefer.